user_docs/en/changes.md

```diff
@@ -2,48 +2,85 @@
 
 ## 2025.1
 
+This release introduces NVDA Remote, allowing you to control a remote copy of NVDA from another device running NVDA.
```

Comment:
```suggestion
This release introduces NVDA Remote, allowing you to control a remote computer from another device running NVDA.
```

Reply:
>Imo, perhaps it's clearer to reflect that another computer can be controlled, to indicate that not just NVDA, but other components of the system, can be controlled.


I'm worried this may be more misleading, implying you can control a remote computer without NVDA

```suggestion
This release introduces NVDA Remote, allowing you to control a remote computer running NVDA from another device running NVDA.
```